frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx

```diff
@@ -211,9 +211,6 @@ export const SidePanelSupport = (): JSX.Element => {
                                     </LemonBanner>
                                 </Section>
                             ) : null}
-
-                            <Section title="Contact us">
-                                <p>Can't find what you need in the docs?</p>
```

Comment:
Why are we removing the section title here? We would need to remove the closing `</Section>` as well, but I guess this is a merge error?

Reply:
Yeah, this is a merge error. Let me just close this and open a new one.